File: fintechplatform-ui/src/main/java/com/fintechplatform/ui/enterprise/db/enterprise/EnterpriseDB.java
package com.fintechplatform.ui.enterprise.db.enterprise;

import com.raizlabs.android.dbflow.sql.language.SQLite;

/**
 * Created by ingrid on 26/02/18.
 */

public class EnterpriseDB {
    public Enterprise findEnterprise(String ownerId){
        return SQLite.select()
                .from(Enterprise.class)
                .where(Enterprise_Table.id.is(ownerId))
                .querySingle();
    }

    public void deleteEnterprises() {
        SQLite.delete().from(Enterprise.class).execute();
    }

    public void saveEnterprise(Enterprise enterprise) {
        enterprise.save();
    }
}

File: fintechplatform/src/main/java/com/fintechplatform/android/enterprise/db/enterprise/EnterpriseDB.kt
package com.fintechplatform.android.enterprise.db.enterprise

import com.raizlabs.android.dbflow.sql.language.SQLite

class EnterpriseDB {
    fun findEnterprise(): Enterprise? {
        return SQLite.select().from(Enterprise::class.java).querySingle()
    }

    fun deleteEnterprise() {
        SQLite.delete().from(Enterprise::class.java).execute()
    }

    fun saveEnterprise(enterprise: Enterprise) {
        enterprise.save()
    }
}